cypress/e2e/with_mock_data/catalogueItems.cy.ts

```diff
@@ -404,6 +405,17 @@ describe('Catalogue Items', () => {
     cy.findByText('Cameras 4').should('exist');
   });
 
+  it('navigates to the landing page, opens the edit dialog, and correctly shows recently added section', () => {
+    cy.clock(new Date("2024-01-09T12:00:00.000+00:00"), ['Date']);
+    
+    cy.findByText('Cameras 1').click();
+    cy.findByRole('button', { name: 'Edit' }).click();
+    cy.findByLabelText('Manufacturer *').click();
+    cy.contains('A-Z').should('be.visible');
+    cy.contains('Recently Added').should('be.visible');
+    cy.findAllByText('Manufacturer B').should('have.length', 2);
+  })
+
```

Comment:
Can you restore the after the test has finished please. https://docs.cypress.io/api/commands/clock#Restore-clock

Reply:
Is it fine to restore it in the afterEach section, for the same reason mentioned here? https://github.com/ral-facilities/inventory-management-system/pull/892#discussion_r1742284330

Reply:
As its only needed for this test alone, I would only restore in this test and mock it in this test instead of using the beforeEach, the same goes for the unit tests.